src/_guides/language/sound-dart.md

```diff
@@ -11,20 +11,27 @@ This guide tells you why and how to write sound (type safe) Dart code.
 You'll learn how to use strong mode to enable soundness, as well as
 how to substitute types safely when overriding methods.
 
+_**Strong mode**_ is a sound static
+type system that uses a combination of static and runtime checks to
+ensure your code is type safe&mdash;that you can never see a value
+whose runtime type does not match its static type.
+With strong mode enabled (in an implementation that has both the
+static and runtime checks), Dart is a sound language.
+
 <aside class="alert alert-info" markdown="1">
-  The terms **sound** Dart, **strong mode** Dart, and **type safe** Dart
-  are sometimes used interchangeably. _Strong mode_ is a sound static
-  type system that uses a combination of static and runtime checks to
-  ensure your code is type safe&mdash;that you can never see a value
-  whose runtime type does not match its static type.
-  With strong mode enabled (in an implementation that has both the
-  static and runtime checks), Dart is a sound language.
+#### Terminology
+
+  - The terms **sound** Dart, **strong mode** Dart, and **type safe** Dart
+    are sometimes used interchangeably.
+  - **Classic** Dart refers to Dart before soundness was added to the language.
+</aside>
+
+<aside class="alert alert-warning" markdown="1">
+  **Warning:**
   Currently, the Dart dev compiler
   ([dartdevc,]({{site.webdev}}/tools/dartdevc) also known as _DDC_)
   is the only full implementation of strong mode.
   VM and dart2js support are on their way.
```

Comment:
"on their way" could link to tracking bugs.

Reply:
I've searched through the SDK issues but didn't find anything appropriate. Maybe you can ask around?